logger/fleet_commander_logger.py

```diff
@@ -1213,13 +1213,16 @@ def __init__(self, connmgr):
 
     def submit_config_change(self, id, data):
         # Prepare data:
-        deserialized_data = json.loads(data)
-        payload_data = {
-            'Title': deserialized_data['title'],
-            'URL': deserialized_data['url'],
-            'Placement': deserialized_data['placement'],
-            'Folder': deserialized_data['folder'],
-        }
+        if data is None:
```

Comment:
To avoid the need to change the way fleet commander admin stores the profile info and the client deploys it we can just only send the config changes if the data is not none.